ml-workflow-demo-kfp-katib-mlflow/README.md

```diff
@@ -0,0 +1,472 @@
+
+# ML Workflow Demo: Kubeflow - Katib - ML Flow
+
+## Overview
+
+This guide intended to introduce end users to complete ML workflow using Kubeflow. In particular, examples of Kubeflow pipeline using Katib hyperparameter tuning and ML Flow model registry are presented along with some common pipeline steps and interfaces such as S3.
+
+The following diagram outlines ML workflow presented in this guide. Major pipeline steps include:
+- Ingestion of dataset.
+- Cleaning up the dataset.
+- Store of cleaned data to S3 bucket.
+- Hyperparameter tuning using Katib and Tensorflow training container image (with ML Flow store functionality).
+- Converting Katib results to streamlined format.
+- Model training using tuning results.
+- Storing the resulting production model to ML Flow model registry.
+
+![Diagram](./images/ML-Workflow-Demo-Diagram.png)
+
+This repository contains all artifacts needed to support this guide. `images/` directory contains all related screenshorts and diagrams. `resources/` directory contains Jupyter notebook containing all steps in this guide, `Dockerfile` and Python script for training image used in this guide.
+
+## Prerequisites
+
+- Deployed Kubeflow instance and access to Kubeflow dashboard. For sample Kubeflow deployment refer to https://charmed-kubeflow.io/docs/quickstart
+- Deployed MLFlow. For deployment of Charmed MLFlow refer to https://charmed-kubeflow.io/docs/mlflow
+- Familiarity with Python, Docker, Jupyter notebooks.
+
+## Instructions
+The following are the instructions that outline the workflow process.
+
+1. Access Kubeflow dashboard via URL, eg http://10.64.140.43.nip.io/
+
+2. Navigate to Notebooks.
+
+3. Create a new notebook.
+  a. Fill in name
+  b. Select Tensorflow image `jupyter-tensorflow-full:v1.6.0`
+  c. Select minimum configuration: 1 CPU and 4GB of RAM
+
+![NotebookCreate](./images/ML-Workflow-NotebookCreate-diag.png)
+
+![NewNotebook](./images/ML-Workflow-NewNotebook-diag.png)
+
+4. Connect to the newly created notebook.
+
+5. Create a Jupyter notebook to hold code that will specify the Kubeflow pipeline.
+
+![NewJupyterNotebook](./images/ML-Workflow-NewJupyterNotebook-diag.png)
+
+NOTE: The following Jupyter notebook contains all the steps outlined below: [ml-workflow-demo-kfp-katib-mlflow.ipynb](./resources/ml-workflow-demo-kfp-katib-mlflow.ipynb)
```

Comment:
Is this note necessary?  I don't see the file it references.  Or is the idea that, if you're reading the README.md, you now have a link to the original ipynb?  If so, I think you just need to remove the leading `./resources/` from the link